examples/DotNetCore/EventHub/EventContract.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using Newtonsoft.Json;
+
+namespace SimpleAppConfigEventHub
+{
+    class EventContract
```

Comment:
>EventContract [](start = 10, length = 13)

This should be KeyValueEvent or ConfigurationSettingEvent. something relating it to Azure App Configuration.

Reply:
Renamed it.